Cryptocurrency Converter/cryptozor.py

```diff
@@ -0,0 +1,37 @@
+import requests
+
+class Cryptozor:
+    def __init__(self, currency, cryptocurrency):
+        self.currency = currency.upper()
+        self.cryptocurrency = cryptocurrency.upper()
+
+    def convert(self, amount):
+        api = requests.get('https://api.coinbase.com/v2/exchange-rates?currency='+self.cryptocurrency)
+        try:
+            currentPrice = api.json()['data']['rates'][self.currency]
+        except KeyError:
+            pass
+        # Bitcoin
+        if self.cryptocurrency == ('BTC'):
```

Comment:
instead of writing multiple if else statement make your code more user friendly with dictionary or write  simple function to do repeated work.

```suggestion
        operations = ['ETH','BCH','ETC','LTC'...]
        if self.cryptocurrency in operations:
           return amount / float(currentPrice)
        else:
           return 'Not Impleted'
```

Reply:
@chavarera I have updated the file cryptozor.py